src/ucs/datastruct/bitmap.h

```diff
@@ -163,72 +159,36 @@ static inline bool _ucs_bitmap_is_zero(const void *bitmap, size_t words)
     typedef struct { \
         ucs_bitmap_word_t bits[_UCS_BITMAP_BITS_TO_WORDS(_length)]; \
     } ucs_bitmap_t(_length); \
-\
-    static inline bool ucs_bitmap_##_length##_is_zero(ucs_bitmap_t(_length) \
-                                                          bitmap) \
-    { \
-        return UCS_BITMAP_IS_ZERO(bitmap); \
-    } \
-\
-    /** \
-     * Perform bitwise NOT of a bitmap \
-     * \
-     * @param _bitmap Negate this bitmap \
-     * \
-     * @return A new bitmap, which is the negation of the given one \
-     */ \
+    \
```

Comment:
how does it work to declare `ucs_bitmap_t(_length)` ? can it compile from C code?
should be ucs_bitmap##_length##_t
or add macro
`#define UCS_BITMAP_TYPE(_length) ucs_bitmap##_length##_t`